src/components/LMap.vue

```diff
@@ -210,9 +215,17 @@ export default {
     };
 
     onMounted(async () => {
-      const { map, CRS, Icon, latLngBounds, latLng, DomEvent } = await import(
-        "leaflet/dist/leaflet-src.esm"
-      );
+      WINDOW_OR_GLOBAL.L = WINDOW_OR_GLOBAL.L || (await import("leaflet"));
+      const {
```

Comment:
Can we restore the formatting originally used? I am assuming you have format on save, in your editor